code/src/Components/TaskList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { TaskItem } from './TaskItem'
+import { TaskInput} from './TaskInput'
+import { useSelector } from 'react-redux'
+
+
+
+export const TaskList = () => {
+const list = useSelector((store) => store.todos.list)
+  
+  return (
+    <section>
+      <TaskInput />
+      {list.items.map((item, index) => {
+       return <TaskItem 
+       key="taskNum"
```

Comment:
Keys should be unique in React. The `item`'s id would be a perfect thing to use for the key here - `key={item.id}`

More info: https://reactjs.org/docs/lists-and-keys.html